airbyte-cdk/python/airbyte_cdk/sources/declarative/stream_slicers/datetime_stream_slicer.py

```diff
@@ -27,12 +28,12 @@ class DatetimeStreamSlicer(StreamSlicer):
     # FIXME: timezone should be declarative?
     def __init__(
         self,
-        start_time: InterpolatedString,
-        end_time: InterpolatedString,
-        step,
-        cursor_value: InterpolatedString,
-        datetime_format,
-        config,
+        start_time: Union[InterpolatedString, str],
+        end_time: Union[InterpolatedString, str],
```

Comment:
just noting that these will probably get wiped as we lock this down to `MinMaxDatetime` depending on which PR gets merged first